apps/vendor-portal/src/components/notification-dropdown.vue

```diff
@@ -169,9 +177,23 @@ export default defineComponent({
       return "#87b563";
     };
 
+    const handleClick = (notifyType: string) => {
```

Comment:
Why do not just emit "notification:click" without any logic for checking the type?